code/game/objects/items/devices/traitordevices.dm

```diff
@@ -131,7 +130,7 @@ effective or pretty fucking useless.
 	popup.open()
 
 /obj/item/device/rad_laser/Topic(href, href_list)
-	if(!in_range(src, usr) || issilicon(usr) || !usr.canmove || usr.restrained())
+	if(!usr.canmove || usr.stat || usr.restrained() || !in_range(loc, usr))
```

Comment:
Yup: https://github.com/ParadiseSS13/Paradise/blob/fe2a3ef960297c6f9a15f4114c2c60634372ad2e/code/game/objects/objs.dm#L24. That uses the Topic checks handled in NanoUI module files.